### PR TITLE
Adds script for Google Analytics

### PR DESCRIPTION
**Pull request by GerardoGomezG**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,6 +11,14 @@
...
         <title>Gerardo's Portfolio</title>
     </head>
+    <!-- Google tag (gtag.js) -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-9E36WY2R8E"></script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+        gtag('config', 'G-9E36WY2R8E');
+    </script>
```
Comment (danyhoshi) [inline index.html]: # Great!!! I suggest locating the script into head tags, like Google Analytics indicate

googleAnalyctics

**Review by danyhoshi — CHANGES_REQUESTED** [review]
# Great!!! 😄  I suggest locating the script into head tags like google analytics indicate

File: index.html (hunk 1 above)
Comment (GerardoGomezG) [reply index.html]: Changes applied!

Thanks...

**Review by danyhoshi — APPROVED** [review]
# Good job! 👍 Let's merge!

**Pull request merged**